apps/vr-tests-react-components/src/stories/Card.stories.tsx

```diff
@@ -75,9 +110,73 @@ storiesOf('Card Converged', module)
       includeHighContrast: true,
       includeDarkMode: true,
     },
-  );
+  )
+  .addStory('scale', () => {
+    const styles = useScaleStyles();
```

Comment:
ooooof no this is the biggest trap ever, i know this because I have fallen so deep into it before

render functions **are not components** and when hooks are used inside `addStory` render functions they will not function as expected. IN this case contexts will never be resolved and always use default values.

If you want to calla hook, just extract the whole thing to an actual react component

Reply:
Thanks for the reminder, let me know if the changes are good